chronicle/release/releasers/github/summarizer.go

```diff
@@ -36,10 +36,26 @@ type Config struct {
 }
 
 type Summarizer struct {
-	git      git.Interface
-	userName string
-	repoName string
-	config   Config
+	git            git.Interface
+	userName       string
+	repoName       string
+	config         Config
+	releaseFetcher releaseFetcher
+}
+
+// changeScope is used to describe the start and end of a changes made in a repo.
+type changeScope struct {
+	Commits []string
+	Start   changePoint
+	End     changePoint
+}
+
+// changePoint is a single point on the timeline of changes in a repo.
+type changePoint struct {
+	Ref       string
+	Tag       *git.Tag
+	Inclusive bool
```

Comment:
I want to make sure I understand here. `Inclusive` means "the changelog should include this point"? And then below, when we generate a changelog that includes the first commit in the repo, we need to emit an inclusive change point (because the first commit is part of the first release), but subsequent releases, where we're asking for something like "commits since the tag v0.5.1", we should make an _exclusive_ range. Is that right?